pkg/web/context_keys.go

```diff
@@ -20,3 +21,14 @@ func UserFromContext(ctx context.Context) (*User, bool) {
 func NewContextWithUser(ctx context.Context, user *User) context.Context {
 	return context.WithValue(ctx, userKey, user)
 }
+
+// IsAuthorized returns whether the request has been authorized
+func IsAuthorized(ctx context.Context) bool {
+	_, ok := ctx.Value(isAuthorizedKey).(bool)
+	return ok
+}
+
+// WithAuthorizedContext sets the boolean flag isAuthorized in the request context
+func WithAuthorizedContext(ctx context.Context) context.Context {
```

Comment:
Would it make sense to rename _WithAuthorizedContext_ to _NewAuthorizedContext_ like it is with the Authentication above -> _NewContextWithUser_?

Or more so renaming the _NewContextWithUser_ function to _NewAuthenticatedContext_ as well?